create_test_files.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
```

Comment:
This is a script I used to update test files with current versions of the scientific libraries. Excuse the messy code– it's only run manually, and meant to make it clear how the test files were generated!

Reply:
Probably makes sense to put it into some folder like /utils or similar and not leave in the root

Reply:
This file in general - I'm not sure you want it to be kept in the repo.
If it's for internal use only - remove it from public repo.
If you reference it in some way in the docs and want users to see it - then you need to treat it as part of the codebase with the similar high bar for the quality :)

Reply:
I have cleaned it significantly, documented it more clearly, and moved it to a utils file :)